src/RemoteTech/Modules/ProtoSignalProcessor.cs

```diff
@@ -25,16 +26,18 @@ public ProtoSignalProcessor(ProtoPartModuleSnapshot ppms, Vessel v)
             mVessel = v;
             Powered = ppms.GetBool("IsRTPowered");
 
+	    int commandMinCrew = DEFAULT_COMMAND_MINIMUM_CREW;
+			
             try {
-                IsCommandStation = Powered && v.HasCommandStation() && v.GetVesselCrew().Count >= ppms.GetInt("RTCommandMinCrew");
+                commandMinCrew = ppms.GetInt("RTCommandMinCrew");
                 RTLog.Notify("ProtoSignalProcessor(Powered: {0}, HasCommandStation: {1}, Crew: {2}/{3})", 
-                    Powered, v.HasCommandStation(), v.GetVesselCrew().Count, ppms.GetInt("RTCommandMinCrew"));
+                    Powered, v.HasCommandStation(), v.GetVesselCrew().Count, commandMinCrew));
             } catch (ArgumentException) {
                 // I'm assuming this would get thrown by ppms.GetInt()... do the other functions have an exception spec?
-                IsCommandStation = false;
-                RTLog.Notify("ProtoSignalProcessor(Powered: {0}, HasCommandStation: {1}, Crew: {2})", 
+                RTLog.Notify("ProtoSignalProcessor(Powered: {0}, HasCommandStation: {1}, Crew: {2}/6 [Default]", 
```

Comment:
you have {2}/6 hard coded in here. you should at least reference your constant from above otherwise if that changes we wont see it in the message. but really it should be reading that value out of the cfg.